### PR TITLE
Update linux recipes

### PR DESCRIPTION
**Pull request by linusnielsen**
The linux-yocto recipe is obsolete, so I removed it. Then I added a linux-intel recipe to build the altera-tse driver.

### CODE REVIEW

**Comment by TSAnsari** [comment]
Ok, I will make a build test with these changes and once it passes, this can be delivered.

**Review by TSAnsari — APPROVED** [review]
The test build worked. So you can deliver this.

File: recipes-kernel/linux/linux-intel/altera-tse.cfg [hunk 1]
```diff
@@ -0,0 +1 @@
+CONFIG_ALTERA_TSE=m
```
Comment (thenor) [inline recipes-kernel/linux/linux-intel/altera-tse.cfg:1]: Would it be possible to add some information on what this means? :smiley:

**Comment by thenor** [comment]
Please start using the s-flag with "git commit" so the commits are signed off. According to our contributing guidelines.

**Review by thenor — CHANGES_REQUESTED** [review]
Update the commit messages with the signed-off info and I will be happy.

**Comment by linusnielsen** [comment]
One force-push later, the commits now have a Signed-off-by line.

**Review by thenor — APPROVED** [review]
Thanks!

**Pull request merged**